t/rest_server/main.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements a simple HTTP/JSON REST example.
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/abcxyz/pkg/renderer"
+	"github.com/go-chi/chi/v5"
+)
+
+var bind = flag.String("bind", ":8080", "Specifies server ip address and port to listen on.")
+
+func handleHello(h *renderer.Renderer) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		h.RenderJSON(w, http.StatusOK, map[string]string{"message": "hello world"})
```

Comment:
Is this an opportunity to use a replacement to show the power of the CLI?

Reply:
I actually advised against doing this; that seemed a bit like complexity for its own sake. We should limit the data we need from users to the minimum to streamline the user experience.

As we evolve this template and add features (auth, DB) then there will be plenty of opportunities to demonstrate our replacement powers.

Reply:
Will leave as no substitution for now.